packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md

```diff
@@ -6,16 +6,16 @@ Tokens contain claims that are statements about the subject (for example: name,
 
 Create ID Token claims for OpenID Connect or access tokens for OAuth 2.0:
 
-1. In the Admin Console, navigate to **Security > API**.
-2. On the **Authorization Servers** tab, select the name of the authorization server, and then click **Claims**. Okta provides a default subject claim. You can edit the mapping or create your own claims.
-3. Click **Add Claim**, enter a **Name** for the claim, and configure the claim settings:
+1. In the Admin Console, go to **Security** > **API**.
+1. On the **Authorization Servers** tab, select the name of the authorization server, and then click **Claims**. Okta provides a default subject claim. You can edit the mapping or create your own claims.
+1. Click **Add Claim**, enter a **Name** for the claim, and configure the claim settings:
 
-   * **Include in token type** &mdash; Select **Access Token** (OAuth 2.0) or **ID Token** (OpenID Connect). If you choose **ID Token**, you can also define whether you want the claim included only when requested or always included.
+    * **Include in token type** &mdash; Select **Access Token** (OAuth 2.0) or **ID Token** (OpenID Connect). If you choose **ID Token**, you can also define whether you want the claim included only when requested or always included.
     * **Value type** &mdash; Select whether you want to define the claim by a **Groups** filter or by an **Expression** written using the Okta Expression Language.
-        * **Value** &mdash; This option appears if you chose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles. Add the mapping here using the [Okta Expression Language](/docs/reference/okta-expression-language/), for example `appuser.username`.
-          Be sure to check that your expression returns the results expected. You can validate an expression using the **Token Preview** tab.
-        * **Filter** &mdash; This option appears if you chose **Groups**. Use it to add a group filter.
-        > **Note:** Up to 100 groups are included in the claim. If the filter results in more than that, the request fails.
+    * **Value** &mdash; This option appears if you chose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles. Add the mapping here using the [Okta Expression Language](/docs/reference/okta-expression-language/), for example `appuser.username`.
```

Comment:
```suggestion
    * **Value** &mdash; This option appears if you chose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles. Add the mapping here using the [Okta Expression Language](/docs/reference/okta-expression-language/), for example, `appuser.username`.
```

Reply:
The start of the description is always capitalized (here and elsewhere). Not sure the extra comma is needed here.